portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,119 @@
 
 package com.google.sps.servlets;
 
+
+import java.util.*;
+import com.google.sps.data.Comment;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
+//the link inside webservelet determines the URL that loads what data
+
+
+
+
+/**--------------------------------------------allows for comment implememntation----------------------------*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+
+	//use query to access data in datastore 
```

Comment:
nit.  For comments they should have a space and start with caps, for example this should be "// Use query to access data in datastore".  Please change here and the other times it happens.